content/root/quick-start-guide.textile

```diff
@@ -343,8 +354,11 @@ The following example code publishes a message to the @quickstart@ channel with
 bc[jsall](code-editor:quick-start-guide/send-message). var channel = ably.channels.get('quickstart');
 channel.publish('greeting', 'hello!');
 
-bc[java]. Channel channel = ably.channels.get('quickstart');
-channel.publish('greeting', 'hello!');
+bc[java]. Channel channel = ably.channels.get("quickstart");
+channel.publish("greeting", "hello", new CompletionListener() {
+  @Override
+  public void onSuccess() {
+    System.out.println("Message sent");
```

Comment:
This code sample is incomplete. I believe it should read:

```
channel.publish("greeting", "hello", new CompletionListener() {
  @Override
  public void onSuccess() {
    System.out.println("Message sent");
  }. // <--
}); // <--
```

Reply:
Ah, this was my mistake - I missed the error handling section here as well as the closing brackets.